packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -114,6 +114,69 @@ The `value` object is where you specify the specific operation to perform. It is
 | replace | Modify an existing claim and update the token lifetime. |
 | remove  | Remove an existing claim. |
 
+#### Reserved claims for Token Hooks
+
+Okta defines a number of reserved claims that can't be overridden. When you add a custom claim to a [token](/docs/guides/validate-id-tokens/overview/#id-tokens-vs-access-tokens) or modify a claim, avoid using the following reserved claims:
```

Comment:
I wonder if it might be clearer not to include that link, if it only applies to ID tokens, and the information here is about both ID and access tokens.

Reply:
Yeah, I know what you mean. I asked Venkat about it last week. He wanted the link. I had suggested another one. Let me ping him again about it. Thanks, Moti.